### ISSUE TITLE
Support for Wine and Proton

### ISSUE THREAD

**Issue by ArtyIF** [OpenAsar/arrpc#35]
Currently, Windows games running via Wine or Proton don't seem to be detected on Linux. I imagine this can be solved by scanning for all the Wine and Proton processes and looking at their arguments for the EXEs running. However, I'm not well versed with this project enough to fix it myself yet.

**Comment by ArtyIF** [comment]
I might've narrowed down the issue. As it turns out, the EXE launched is already in the output of `ps`. However, the way it's launched in arRPC only shows processes for the current terminal session, not the user. There needs to be an additional `-u <username>` argument. If I'm right, I'll make a PR.

**Comment by CanadaHonk** [comment]
I didn't have that issue. Also, it isn't that simple as arRPC uses `/proc/{pid}/exe` to get the binary path, which will probably result in a Wine path.

**Comment by ArtyIF** [comment]
You're right. I misinterpreted the output. It does link to `wine-preloader`. However, cmdline has the path to the EXE.

![image](https://github.com/OpenAsar/arrpc/assets/8017858/b447cdbc-2d5e-4fed-b025-5a1073dba6a4)

**Comment by cannibalmaid** [comment]
Are you still having this issue? Detecting wine games and such works perfectly for me using my fork. My code is probably messy.
https://github.com/cannibalmaid/arrpc

**Comment by sull324** [comment]
No process are detected using the latest git version but if i use https://github.com/cannibalmaid/arrpc fork everything work fine and is detected.

using latest arch linux.
